packages/react-charting/src/components/HorizontalBarChartWithAxis/__snapshots__/HorizontalBarChartWithAxis.test.tsx.snap

```diff
@@ -178,7 +179,7 @@ exports[`HorizontalBarChartWithAxis snapShot testing renders HorizontalBarChartW
                   &:focus {
                     outline: none;
                   }
-              data-focuszone-id="FocusZone5"
+              data-focuszone-id="FocusZone6"
```

Comment:
add few tests snapshots after enabling gradient for vertical bar, donut and, horizontal bar

Reply:
Added snapshots for gradients and rounded corners for Donut, Vertical, Vertical Stacked, Horizontal and Horizontal Stacked charts